storage/innobase/xtrabackup/test/suites/binlog/binlog_101.sh

```diff
@@ -0,0 +1,13 @@
+. inc/common.sh
+. inc/binlog_common.sh
+# do the same, but with updating my.cnf
+
+vlog "------- TEST 101 -------"
+MYSQLD_EXTRA_MY_CNF_OPTS="
+skip-log-bin
+"
+INDEX_FILE=""
```

Comment:
if index_file is empty, how do we know which binlog index file to use for verification?

Reply:
INDEX_FILE is tested as `-n` which will be skipped on empty values or null